app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -375,7 +373,7 @@ class DescriptionEditFragment : Fragment() {
     }
 
     private fun updateDescriptionInArticle(articleText: String, newDescription: String): String {
-        return if (templateParsePattern.matcher(articleText).find()) {
+        return if (TEMPLATE_PARSE_REGEX.toRegex().matches(articleText)) {
```

Comment:
When making any kind of change to a feature, please try _using_ the feature and make sure it still works.

Reply:
Fixed and done.